examples/compute_head_direction.py

```diff
@@ -0,0 +1,397 @@
+    left_keypoint="left_ear",
+    right_keypoint="right_ear",
+    # Optional parameters:
+    reference_vector=(1, 0),  # positive x-axis
+    camera_view="top_down",
+    in_radians=True,  # set to False for degrees
+)
+print(forward_vector_angle)
+
+# %%
+# The resulting ``forward_vector_angle`` array contains the head direction
+# angle in radians, with respect to the positive x-axis, meaning that
+# the angle is zero when the head is pointing to the right.
+# We could have also used an alternative reference vector, such as the
+# negative y-axis (pointing up in image coordinates) by setting
+# ``reference_vector=(0, -1)``. You may control the sign of the angle
+# by altering the ``ref_to_forward`` parameter, see notes is
+# :func:`compute_forward_vector_angle()\
+# <movement.kinematics.compute_forward_vector_angle>`'s documentation.
+
+# %%
+# Visualise head direction angles
+# -------------------------------
+# We can compare the head direction angles computed from the two methods,
+# i.e. the polar angle ``phi`` of the head-to-snout vector and the angle
+# of the forward vector, by plotting their histograms in polar coordinates.
+# First, let's define a custom plotting function that will help us with this.
+
+
+def plot_polar_histogram(da, bin_width_deg=15, ax=None):
+    """Plot a polar histogram of the data in the given DataArray.
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+        A DataArray containing angle data in radians.
+    bin_width_deg : int, optional
+        Width of the bins in degrees.
+    ax : matplotlib.axes.Axes, optional
+        The axes on which to plot the histogram.
+
+    """
+    n_bins = int(360 / bin_width_deg)
+
+    if ax is None:
+        fig, ax = plt.subplots(  # initialise figure with polar projection
+            1, 1, figsize=(5, 5), subplot_kw={"projection": "polar"}
+        )
+    else:
+        fig = ax.figure  # or use the provided axes
+
+    # plot histogram using xarray's built-in histogram function
+    da.plot.hist(
+        bins=np.linspace(-np.pi, np.pi, n_bins + 1), ax=ax, density=True
+    )
+
+    # axes settings
+    ax.set_theta_direction(-1)  # theta increases in clockwise direction
+    ax.set_theta_offset(0)  # set zero at the right
+    ax.set_xlabel("")  # remove default x-label from xarray's plot.hist()
+
+    # set xticks to match the phi values in degrees
+    n_xtick_edges = 9
+    ax.set_xticks(np.linspace(0, 2 * np.pi, n_xtick_edges)[:-1])
+    xticks_in_deg = (
+        list(range(0, 180 + 45, 45)) + list(range(0, -180, -45))[-1:0:-1]
+    )
+    ax.set_xticklabels([str(t) + "\N{DEGREE SIGN}" for t in xticks_in_deg])
+
+    return fig, ax
+
+
+# %%
+# Now we can visualise the polar ``phi`` angles of the ``head_to_snout_polar``
+# array alongside the values of the ``forward_vector_angle`` array.
+
+# sphinx_gallery_thumbnail_number = 3
+
+head_to_snout_angles = head_to_snout_polar.sel(space_pol="phi")
+forward_vector_angles = forward_vector_angle
```

Comment:
```suggestion
head_to_snout_angle = head_to_snout_polar.sel(space_pol="phi")
```

Is `forward_vector_angles = forward_vector_angle` worth renaming?

Reply:
Not really, you are right.